Gds/src/fprime_gds/common/client_socket/client_socket.py

```diff
@@ -104,12 +106,11 @@ def disconnect(self):
         self.__data_recv_thread.join()
         self.sock.close()
 
-    def data_callback(self, data, sender=None):
+    def data_callback(self, data):
```

Comment:
This argument is important as it fits a "duck" type from elsewhere. I would revert this change.

Reply:
OK, feel free to revert my wrong changes.